config/clusters/callysto/common.values.yaml

```diff
@@ -32,6 +32,28 @@ jupyterhub:
     image:
       name: callysto/2i2c
       tag: 0.1.0
+    extraFiles:
+      tree.html:
+        mountPath: /usr/local/share/jupyter/custom_template/tree.html
+        stringData: |
+          {% extends "templates/tree.html" %}
+          {% block header %}
+          <style>
+          .clusters_tab_link {
+            visibility: hidden;
+          }
+          </style>
```

Comment:
Hmm, actually this one is only possible in "tree.html", so we would have needed two templates anyway